src/azul/project/hca/transformers.py

```diff
@@ -171,6 +201,7 @@ def create_documents(self,
                             version=version,
                             manifest=manifest,
                             metadata_files=metadata_files)
+        # Create ElasticSearch documents
```

Comment:
`Elasticsearch` (lower case s), but honestly, the method is called `create_documents` what does this comment add?